app/code/Bulbulatory/Recommendations/etc/module.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Bulbulatory_Recommendations" setup_version="0.0.1">

```

Comment:
I suggest to use the last part of the version number for bug fixing only. You can read more about versioning in this article: https://en.wikipedia.org/wiki/Software_versioning.

Reply:
@rafdol 

> I suggest to use the last part of the version number for bug fixing only. You can read more about versioning in this article: https://en.wikipedia.org/wiki/Software_versioning.

Thanks for the advice and link. I fixed module versioning.